openspp-project/openspp-registry
# fix issues on updating names and mobile no

## **Why is this change needed?**
To fix the issues when creating names with special character and fix the updating of mobile number

## **How was the change implemented?**
Fixed the create and write functions for groups

## **New unit tests**
```
None
```

## **Unit tests executed by the author**
```
None
```

## **How to test manually**
- Install or Upgrade `spp_farmer_registry_base` and `spp_registrant_import`.

Issue 1:
- Create or Update a Group.
- In Farmer's Tab add a farmer with a special character name (e.g. O'Cronin).
- Save and Check if the name was not change to "O'cronin".

Issue 2:
- Create or Update a Group.
- In Farmer's Tab add a mobile number and Save.
- Try changing the mobile number again to see if it updates the mobile number.

## **Related links**
https://github.com/orgs/OpenSPP/projects/5/views/4?pane=issue&itemId=73820434
